ember/app/gql/queries/get-case.graphql

```diff
@@ -52,7 +52,7 @@ query getCase($filter: [CaseFilterSetType]) {
                         "gesprochener-rahmenkredit"
                         "advance-credit-date"
                         "advance-credit-amount-float"
-                        "advance-credit-remark-staff"
+                        "notiz-intern"
```

Comment:
```suggestion
                        "notizen-intern"
```

Reply:
Still a draft :joy: weird that codeowners get assigned on drafts. But yes, will fix.